src/tests/tf/test_text_preprocessing_layer.py

```diff
@@ -0,0 +1,20 @@
+import tensorflow as tf
+
+from rdds.lib.tf import TextPreprocessingLayer
+
+
+def test_preprocessing_layer():
+
+    preprocessing_layer: TextPreprocessingLayer = TextPreprocessingLayer()
```

Comment:
```suggestion
    preprocessing_layer = TextPreprocessingLayer()
```
No need to type hint when calling a constructor. This is clear enough.